.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
     - ARTIFACTS_UPLOAD_PATH_BRANCH=travis_build_artifacts/${TRAVIS_REPO_SLUG}/${TRAVIS_BRANCH}/${TRAVIS_BUILD_NUMBER}
     - ARTIFACTS_UPLOAD_PATH_PR=travis_build_artifacts_pr/${TRAVIS_REPO_SLUG}/${TRAVIS_BRANCH}/${TRAVIS_BUILD_NUMBER}
     - TEST_FLAGS=""
+    - PRODUCT_TESTS_CONFIG=config-empty
```

Comment:
Might be cleaner to remove the `config-` prefix from the environment vars.

Reply:
i intentionally left this as part of the value, so that value is more searchable.
(I did the same with `PRODUCT_TESTS_SUITE`, btw)